source/draw/Color.ooc

```diff
@@ -133,13 +133,14 @@ ColorRgba: cover {
 	r, g, b, a: Byte
 	normalized ::= FloatTuple4 new(this r as Float / 255, this g as Float / 255, this b as Float / 255, this a as Float / 255)
 	black ::= static This new(0, 0, 0, 255)
+	transparent ::= static This new(0, 0, 0, 0)
 	white ::= static This new(255, 255, 255, 255)
 	red ::= static This new(255, 0, 0, 255)
 	green ::= static This new(0, 255, 0, 255)
 	blue ::= static This new(0, 0, 255, 255)
 
 	init: func@ (=r, =g, =b, =a)
-	init: func@ ~default { this init(0, 0, 0, 0) }
+	init: func@ ~default { this init(0, 0, 0, 255) }
```

Comment:
Having transparent default RGBA made no sense when default RGB, YUV and YUVA is opaque black. The `transparent` color was added for backward compatibility.